examples/linear_model.ipynb

```diff
@@ -0,0 +1,306 @@
+    "* `measurement_noise_dist : Optional, str`\n",
+    "  * distribution for measurement noise (e.g., normal, uniform, triangular)\n",
+    "* `g : Optional, float`\n",
+    "  * Acceleration due to gravity (m/s^2). Default is 9.81 m/s^2 (standard gravity)\n",
+    "* `thrower_height : Optional, float`\n",
+    "  * Height of the thrower (m). Default is 1.83 m\n",
+    "* `throwing_speed : Optional, float`\n",
+    "  * Speed at which the ball is thrown (m/s). Default is 40 m/s"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With our definitions, we can now create the ThrownObject Model.\n",
+    "\n",
+    "First, we would need to import the necessary packages."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from prog_models import LinearModel"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we'll define some features of ThrownObject such that it models after a LinearModel. Remember at the top of the Jupyter Notebook, there were a few properties that all Linear Models must define? In the next step, we'll define our inputs, states, outputs, and events, along with the $A$, $C$, $E$, and $F$ values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ThrownObject(LinearModel):\n",
+    "    events = ['impact']\n",
+    "    inputs = []  \n",
+    "    states = ['x', 'v']\n",
+    "    outputs = ['x']\n",
+    "    \n",
+    "    A = np.array([[0, 1], [0, 0]])\n",
+    "    C = np.array([[1, 0]])\n",
+    "    E = np.array([[0], [-9.81]])\n",
+    "    F = None"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll define some default parameters for our ThrownObject model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class ThrownObject(ThrownObject):  # Continue the ThrownObject class\n",
+    "    default_parameters = {\n",
+    "        'thrower_height': 1.83,\n",
+    "        'throwing_speed': 40,\n",
+    "        'g': -9.81\n",
+    "    }"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the following cells, we'll define some necessary class functions such that we are able to perform prognostics on the model!"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `initialize()` function sets the states to their corresponding values. E.g., we have defined that our `x` value for our ThrownObject model to represent position in space, therefore, our initial value for the position in space would be the thrower_height parameter!"
```

Comment:
```suggestion
    "The `initialize()` function sets the states to their corresponding values. Since we have defined the `x` value for our ThrownObject model to represent position in space, our initial value would be the thrower_height parameter."
```